.scalafix.conf

```diff
@@ -1,4 +1,10 @@
 rules = [
   RemoveUnused,
   LeakingImplicitClassVal
+  ProcedureSyntax
+  ExplicitResultTypes
 ]
+
+ExplicitResultTypes.memberKind = [Def, Val, Var]
```

Comment:
```suggestion
ExplicitResultTypes.memberKind = [Def, Var]
```

😁

Reply:
ahah 😄 I don't want to turn it off for `Val` there are cases where sometimes it might not be immediately obvious i.e `val schema: Schema = Schema.createRecord("GenericTestRecord", null, null, false)` and all the `TapT[T]` (which we should probably revise at some point).

I'm still looking into this because I get the feeling that `ExplicitResultTypes.skipSimpleDefinition` is not working as i expected ... I was unable to remove those `val s: String = "foo"` but i have the feeling that it should be possible

Reply:
turns out that if we simply set `ExplicitResultTypes.skipSimpleDefinition = false` it behaves a lot more as we wanted it to.